### ISSUE TITLE
Login Page

### ISSUE THREAD

**Issue by girlscriptfoundation** [girlscriptfoundation/communityportal#1]
Login page will have Sign up and Sign in options. 
If you are a chapter leader, sign up as a leader.
If you are a volunteer, sign up as a volunteer.
If you are a member or would like to become a member, sign up for membership.

**Comment by rounaksarda** [comment]
Can i share the webpage of Sign in and Sign up

**Comment by girlscriptfoundation** [comment]
Yes @rounaksarda, we would appreciate people who would take a lead and start sharing their ideas here! 😄
